src/shogun/base/base_types.h

```diff
@@ -48,7 +49,8 @@ namespace shogun
 	                    std::is_same<CDifferentiableFunction, T>::value ||
 	                    std::is_same<CInference, T>::value ||
 	                    std::is_same<CLikelihoodModel, T>::value ||
-	                    std::is_same<CMeanFunction, T>::value>
+	                    std::is_same<CMeanFunction, T>::value ||
+	                    std::is_same<CKernelNormalizer, T>::value>
```

Comment:
@gf712 I was trying here to add an sg_type_map for Shogun objects. I am not sure if I am going to need it in the end.